pkg/0e89154b-a448-4fea-a115-169fb56c0cb9/package.lp

```diff
@@ -0,0 +1,185 @@
+(librepcb_package 0e89154b-a448-4fea-a115-169fb56c0cb9
+ (name "SOT353")
+ (description "aka SC70-5, SC88A, TSSOP-5, UMT5\n\nhttps://en.wikipedia.org/wiki/Small-outline_transistor")
+ (keywords "sot353,sc70-5,sc88a,tssop-5,umt5")
+ (author "Danilo B.")
+ (version "0.1")
+ (created 2018-11-01T19:29:35Z)
+ (deprecated false)
+ (category b77c2754-fc91-4e9e-949e-5568a4bee0fe)
+ (pad 34f43b73-db56-4649-8904-0d258eeb7bfe (name "1"))
+ (pad a2b521d2-b58b-45b4-944a-65bd6e6e158a (name "2"))
+ (pad fa0eed07-e0ca-4ed6-bb33-4c54a46bb925 (name "3"))
+ (pad dc18d314-08ec-45f8-bd81-8b83857d1402 (name "4"))
+ (pad a837e245-00a8-45f7-befe-8aaafc1001f8 (name "5"))
+ (footprint 071e06d0-7e9d-4667-8a85-ab3dad74a72a
+  (name "default")
```

Comment:
This is actually the reflow footprint, right? Then you should name it "reflow" to make this clear.

Reply:
Yes. But then it would be consequential to also name the regular footprint variants for all SMD parts (even with just a single variant) "reflow" :slightly_smiling_face:

Reply:
Sure, the reason why other footprints aren't called "reflow" is just because most of them were imported from Eagle, and the importer doesn't know if a footprint is for reflow soldering. So it just calls them "default". Also LibrePCB names them "default" when creating a new package because LibrePCB also doesn't know it better.